wrappers/Mathematica/RefpropLink/PacletInfo.wl

```diff
@@ -3,7 +3,7 @@
 PacletObject[
     <|
         "Name" -> "RefpropLink",
-        "Version" -> "1.1.0",
+        "Version" -> "1.1.1",
         "WolframVersion" -> "13.0+",
         "SystemID" -> {"Windows-x86-64"},
```

Comment:
Note that in PacletInfo.wl, this paclet will only run on "Windows-x86-64" for now, and only on Mathematica 13 or later.